apps/src/templates/studioHomepages/CourseBlocks.jsx

```diff
@@ -153,17 +171,19 @@ export class CourseBlocksHoc extends Component {
     return (
       <div>
         <div className="row">
-          <ProtectedStatefulDiv ref="minecraft"/>
-          <ProtectedStatefulDiv ref="starwars"/>
-          <ProtectedStatefulDiv ref="frozen"/>
-          <ProtectedStatefulDiv ref="hourofcode"/>
+          <ProtectedStatefulDiv ref="0"/>
+          <ProtectedStatefulDiv ref="1"/>
+          <ProtectedStatefulDiv ref="2"/>
+          <ProtectedStatefulDiv ref="3"/>
         </div>
 
         {this.props.rowCount > 1 && (
           <div>
             <br/>
             <br/>
             <div className="row">
+              {/* TODO: (madelynkasula) If Flappy is in 1st row, what should replace Flappy on line below? */}
+              {/* Can we remove 2nd row? It's never used... */}
```

Comment:
Any resolution to this yet? I'm curious.

Reply:
no, i totally forgot--thank you for reminding me! 😂 i'll talk to poorva and address in a follow-up